src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -335,6 +349,52 @@ function attachInteropInvoker(): void {
   });
 }
 
+async function loadTimezone(timeZoneResource: LoadingResource) : Promise<void> {
+  const runDependencyId = `blazor:timezonedata`;
+  Module.addRunDependency(runDependencyId);
+  const request = await timeZoneResource.response;
+  let arrayBuffer = await request.arrayBuffer();
+
+  // The contents of the TZ file look like so
```

Comment:
Here's where we produce the file: https://github.com/dotnet/blazor/pull/2024/files#diff-4a3685061a6b477a3024cb4205ed3a4fR43-R47. Please let me know if I'm missing something important.

Reply:
The implementation looks great! Suggestions:

* In the comment block, mention that we are responsible for generating this file, and link to our code in the other repo that does it. Otherwise it looks like it's some sort of industry-standard file format.
* Preferably, factor out this file parsing logic (lines 358-391) into a separate file, since `MonoPlatform.ts` is quite a complicated bundle of semi-related things already